### PR TITLE
PLT-398: Buildkite script for evaluation test

### PR DESCRIPTION
**Pull request by zliu41**
For pipeline: https://buildkite.com/input-output-hk/plutus-script-evaluation

### CODE REVIEW

File: build/buildkite/script-evaluation-test.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# A nightly job which downloads script evaluation dumps from S3 and runs a regression test.
+steps:
+  - label: "Download and unzip dump files"
+    command: |
+      nix-shell --run 'LOCAL_DIR=$(realpath .)/../mainnet-script-dump-downloaded ./scripts/s3-sync-unzip.sh s3://plutus/mainnet-script-dump/ \\*.event.bz2'
```
Comment (michaelpj) [inline build/buildkite/script-evaluation-test.yml:5]: Why isn't just passing the relative path enough? This maybe needs some comments about where the dump is on the machine - why is it at that path?

If you could just pass the relative path then you could also pass it with 
```
env:
  LOCAL_DIR: "../mainnet-script-dump-download"
```
which would make the pipeline a bit simpler.

Comment (zliu41) [reply build/buildkite/script-evaluation-test.yml:7]: Passing relative path should be enough - just want to guard against accidental change to the working directory in the scripts.

Added a comment about where the dump is on the machine:

```
# The "../" is the parent of the pipeline-specific directory. This makes the script dump easily accessible
# regardless of which buildkite pipeline we use.
```

**Review by michaelpj — APPROVED** [review]
I think it's fine, just unsure about whether this is actually a path we can rely on being able to write to/being persistent.

File: build/buildkite/script-evaluation-test.yml (hunk 1 above)
Comment (zliu41) [reply build/buildkite/script-evaluation-test.yml:7]: It is, but good point about $HOME. I'll make a note here.

Comment (zliu41) [reply build/buildkite/script-evaluation-test.yml:8]: Yeah, I think $HOME is better. I'll update it.

**Review by michaelpj — APPROVED** [review]

**Pull request merged**